packages/core/src/tracing/errors.ts

```diff
@@ -29,3 +29,7 @@ function errorCallback(): void {
     activeTransaction.setStatus(status);
   }
 }
+
+// The function name will be lost when bundling but we need to be able to identify this listener later to maintain the
+// node.js default exit behaviour
+errorCallback.name = 'tracingErrorCallback';
```

Comment:
```suggestion
errorCallback.name = 'sentry_tracingErrorCallback';
```

This way we can make sure there will never be colliding names here.